src/components/page/LittlestarLastTimePodcastSection.tsx

```diff
@@ -112,20 +127,69 @@ const LittlestarLastTimePodcastSection: React.VFC<{
   )
 }
 
-const useLatestPodcast: () => {
-  podcast: {
-    coverUrl: string
+const useLastWatchedPodcastAlbumPodcastProgram: (memberId: string) => {
+  status: 'loading' | 'error' | 'success' | 'idle'
+  lastWatchedPodcastAlbumPodcastProgram: {
+    id: string
     title: string
-    programTitle: string
+    coverUrl: string | null
     categoryNames: string[]
-  }
-} = () => {
+    podcastAlbum: {
+      id: string
+      title: string
+    }
+  } | null
+} = memberId => {
+  const { loading, error, data } = useQuery<
+    hasura.GET_LAST_WATCHED_PODCAST_PROGRAM,
+    hasura.GET_LAST_WATCHED_PODCAST_PROGRAMVariables
+  >(
+    gql`
+      query GET_LAST_WATCHED_PODCAST_PROGRAM($memberId: String!) {
+        podcast_program_progress(where: { member_id: { _eq: $memberId } }, order_by: { updated_at: desc }, limit: 1) {
+          id
+          progress
+          last_progress
+          podcast_program {
+            id
+            title
+          }
+          podcast_album {
+            id
+            title
+            cover_url
+            podcast_album_categories {
+              id
+              category {
+                id
+                name
+              }
+            }
+          }
+        }
+      }
+    `,
+    {
+      variables: { memberId },
+      fetchPolicy: 'no-cache',
+    },
+  )
+  const [lastWatchedPodcastAlbumPodcastProgram = null] = data?.podcast_program_progress || []
   return {
-    podcast: {
-      coverUrl: 'https://static.kolable.com/images/littlestar/podcast-cover3.png',
-      title: '正確的洗手',
-      programTitle: '第 35 期 - 對抗病毒大作戰 第 4 則',
-      categoryNames: ['親子', '公衛防疫'],
+    status: loading ? 'loading' : error ? 'error' : data ? 'success' : 'idle',
+    lastWatchedPodcastAlbumPodcastProgram: lastWatchedPodcastAlbumPodcastProgram && {
+      id: lastWatchedPodcastAlbumPodcastProgram.podcast_program.id,
+      lastProgress: lastWatchedPodcastAlbumPodcastProgram.last_progress,
+      title: lastWatchedPodcastAlbumPodcastProgram.podcast_album?.title || '',
```

Comment:
I except this is pocastProgram's title by naming `last...PodcastProgram`, but this is album's title